ts/Dashboards/Components/Component.ts

```diff
@@ -1207,9 +1212,18 @@ namespace Component {
 
         /**
          * Cell id, where component is attached.
+         *
+         * @deprecated
          */
         cell?: string;
 
+        /**
+         * Cell id, where component is attached.
+         *
+         * @deprecated
```

Comment:
Why this is deprecated?

Reply:
Because we will use `renderTo` however we leave the `cell` option to support back-compatybility. I can imagine that in the future we remove it. Am I right @stitot, or I misunderstood something?

Reply:
Yes, that's my take on it at least